server/channels/api4/hosted_customer.go

```diff
@@ -16,40 +14,9 @@ import (
 func (api *API) InitHostedCustomer() {
 	// POST /api/v4/hosted_customer/available
 	api.BaseRoutes.HostedCustomer.Handle("/signup_available", api.APISessionRequired(handleSignupAvailable)).Methods(http.MethodGet)
```

Comment:
It seems this is no longer used either as it was part of the self-serve purchase - if so, I think we can delete this whole file. Any concerns @nickmisasi ?